modules/persistent-storage-csi-sc-disable.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-sc-disable_{context}"]
+= Disabling the default storage class
+
+Disabling the default storage class allows you to accomplish several different objectives:
+
+* Enforce static provisioning by disabling dynamic provisioning.
+
+* When you have other preferred storage classes, prevent the storage operator from re-creating the initial default storage class.
+
+* Rename the default storage class by deleting the original default storage class, and then creating a new default storage class with a custom name.
+
+.Prerequisites
+* Access to the {product-title} web console.
```

Comment:
You can do that with cli no need to access the web console, i think we should document both. You need admin privileges though. cc @dobsonj

Reply:
FWIW, this is how I change it from the CLI:
```
oc patch clustercsidriver $DRIVERNAME --type=merge -p "{\"spec\":{\"storageClassState\":\"${STATE}\"}}"
```
where `${STATE}` is "Removed" or "Managed" or "Unmanaged"

Reply:
Fixed.